resources/views/account/partials/register-account.blade.php

```diff
@@ -1,65 +1,70 @@
 <x-rapidez::recaptcha location="customer_create"/>
 <graphql-mutation
-    query="mutation customer ($firstname: String!, $lastname: String!, $email: String!, $password: String, $taxvat: String) { createCustomerV2 ( input: { firstname: $firstname, lastname: $lastname, email: $email, password: $password, taxvat: $taxvat } ) { customer { email } } }"
+    query="mutation customer ($firstname: String!, $lastname: String!, $email: String!, $password: String, $taxvat: String, $is_subscribed: Boolean) { createCustomerV2 ( input: { firstname: $firstname, lastname: $lastname, email: $email, password: $password, is_subscribed: $is_subscribed, taxvat: $taxvat } ) { customer { email } } }"
```

Comment:
This is becoming quite a large mutation, there is no syntax highlighting or autocompletion on the query using it this way.

I think it would be good to create a seperate file for the query itself, see:
https://github.com/rapidez/amasty-automatic-related-products/tree/master/resources/views/queries
and include it here, see:
https://github.com/rapidez/amasty-automatic-related-products/blob/master/resources/views/components/productbundles.blade.php#L3

(Also a mutation name of "createCustomer" would be a bit more clear 🙂)